File: auto-docs/executables/julia/axes-labels.jl
using Plotly

using Plotly
Plotly.signin("theengineear", "o9zlr0hy6z")

trace1 = [
  "x" => [0, 1, 2, 3, 4, 5, 6, 7, 8], 
  "y" => [8, 7, 6, 5, 4, 3, 2, 1, 0], 
  "type" => "scatter"
]
trace2 = [
  "x" => [0, 1, 2, 3, 4, 5, 6, 7, 8], 
  "y" => [0, 1, 2, 3, 4, 5, 6, 7, 8], 
  "type" => "scatter"
]
data = [trace1, trace2]
layout = [
  "xaxis" => [
    "title" => "AXIS TITLE", 
    "showticklabels" => true, 
    "tickangle" => 45, 
    "exponentformat" => "e", 
    "showexponent" => "All", 
    "titlefont" => [
      "family" => "Arial, sans-serif", 
      "size" => 18, 
      "color" => "lightgrey"
    ], 
    "tickfont" => [
      "family" => "Old Standard TT, serif", 
      "size" => 14, 
      "color" => "black"
    ]
  ], 
  "yaxis" => [
    "title" => "AXIS TITLE", 
    "showticklabels" => true, 
    "tickangle" => 45, 
    "exponentformat" => "e", 
    "showexponent" => "All", 
    "titlefont" => [
      "family" => "Arial, sans-serif", 
      "size" => 18, 
      "color" => "lightgrey"
    ], 
    "tickfont" => [
      "family" => "Old Standard TT, serif", 
      "size" => 14, 
      "color" => "black"
    ]
  ]
]

response = Plotly.plot([data], ["layout" => layout, "filename" => "axes-labels", "fileopt" => "overwrite", "auto_open" => "false"])
plot_url = response["url"]

File: auto-docs/executables/python/axes_labels.py
# Learn about API authentication here: {BASE_URL}/python/getting-started
# Find your api_key here: {BASE_URL}/settings/api

import plotly.plotly as py
from plotly.graph_objs import *

py.sign_in('TestBot', 'r1neazxo9w')
trace1 = Scatter(
    x=[0, 1, 2, 3, 4, 5, 6, 7, 8],
    y=[8, 7, 6, 5, 4, 3, 2, 1, 0]
)
trace2 = Scatter(
    x=[0, 1, 2, 3, 4, 5, 6, 7, 8],
    y=[0, 1, 2, 3, 4, 5, 6, 7, 8]
)
data = Data([trace1, trace2])
layout = Layout(
    xaxis=XAxis(
        title='AXIS TITLE',
        titlefont=Font(
            family='Arial, sans-serif',
            size=18,
            color='lightgrey'
        ),
        showticklabels=True,
        tickangle=45,
        tickfont=Font(
            family='Old Standard TT, serif',
            size=14,
            color='black'
        ),
        exponentformat='e',
        showexponent='All'
    ),
    yaxis=YAxis(
        title='AXIS TITLE',
        titlefont=Font(
            family='Arial, sans-serif',
            size=18,
            color='lightgrey'
        ),
        showticklabels=True,
        tickangle=45,
        tickfont=Font(
            family='Old Standard TT, serif',
            size=14,
            color='black'
        ),
        exponentformat='e',
        showexponent='All'
    )
)
fig = Figure(data=data, layout=layout)
plot_url = py.plot(fig, filename='axes-labels', auto_open=False)
